content/blog/mapr-for-predictive-maintenance.md

```diff
@@ -0,0 +1,318 @@
+* [RNN predictions for MapR-DB data via Drill](https://github.com/mapr-demos/predictive-maintenance/blob/master/notebooks/jupyter/RNN%20predictions%20on%20MapR-DB%20data%20via%20Drill.ipynb) also shows how to train a model to predict the next value in a sequence of numbers, except it reads time-series data from MapR-DB using Drill as a SQL engine.
+
+## StreamSets Demonstration
+
+To give you an idea of what dataflow management tools do, I’ve prepared a simple StreamSets project that you can run on a laptop with Docker. This project demonstrates a pipeline that streams time-series data recorded from an industrial HVAC system into OpenTSDB for visualization in Grafana. 
+
+Create a docker network to bridge containers:
+
+```console
+docker network create mynetwork
+```
+
+Start StreamSets, OpenTSDB, and Grafana:
+
+```console
+docker run -it -p 18630:18630 -d --name sdc --network mynetwork \
+streamsets/datacollector
+docker run -dp 4242:4242 --name hbase --network mynetwork \
+petergrace/opentsdb-docker
+docker run -d -p 3000:3000 --name grafana --network mynetwork \
+grafana/grafana
+```
+
+Open Grafana at http://localhost:3000 and login with admin / admin
+
+Add http://hbase:4242 as an OpenTSDB datasource to Grafana. If you don’t know how to add a data source, refer to Grafana docs. Your datasource definition should look like this:
+
+<img src="https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/images/grafana_opentsdb_config.png" width="100%" align="center">
+
+Download the following Grafana dashboard file:
+
+```console
+wget https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/Grafana/IoT_dashboard.json
+```
+
+Import that file into Grafana. If you don’t know how to import a dashboard, see Grafana docs. The Grafana import dialog should look like this:
+
+<img src="https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/images/grafana_dashboard_config.png" width="100%" align="center">
+
+Download, unzip, and copy the MQTT dataset to the StreamSets container:
+
+```console
+wget https://github.com/mapr-demos/predictive-maintenance/raw/master/sample_dataset/mqtt.json.gz
+unzip mqtt.json.gz
+docker cp mqtt.json sdc:/tmp/mqtt.json
+```
+
+Open StreamSets at http://localhost:18630 and login with admin / admin
+
+Download and import the following pipeline into StreamSets. If you don’t know how to import a pipeline, refer to StreamSets docs.
+
+```console
+wget https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/StreamSets/MQTT%20File%20Tail.json
+```
+
+You will see a warning about a missing library in the “Parse MQTT JSON” stage. Click that stage and follow the instructions to install the Jython library.
```

Comment:
Is it Jython or is that a typo and should be Python? I know Jython exists... just double checking...

Reply:
Yep, "Jython" is correct